### ISSUE TITLE
Lelll me.alpha432.oyvey riley stop pasting lollzzz

### ISSUE THREAD

**Issue by RealzPrestige** [gerald0mc/geraldhack-1.0#2]
Nice paste riley keep sucking lip balm on gang bdk bdk

**Comment by Doogie13** [comment]
i feel like there is a reason there is no SRC

**Comment by gerald0mc** [comment]
> i feel like there is a reason there is no SRC

yes. there is a reason. its because i dont have it anymore i deleted the folder after i started gerald(Hack) all i have is the jar from releases and never made a git for it

**Comment by RealzPrestige** [comment]
lolllll because you pasted kekekeekekke maddd lelll fold
